deploy/chart/templates/events-enclave-processor-deployment.yaml

```diff
@@ -0,0 +1,38 @@
+{{- range .Values.eventsEnclaveProcessor.instances }}
```

Comment:
@P-Gill97 - This is a little magical, but makes use of [helm's range](https://helm.sh/docs/chart_template_guide/control_structures/#looping-with-the-range-action) to DRY things up. The net result in this case is 3 `Deployment` configurations in the final YAML that it will generate.